esphome/const.py

```diff
@@ -697,6 +697,7 @@
 CONF_STOP = "stop"
 CONF_STOP_ACTION = "stop_action"
 CONF_STORE_BASELINE = "store_baseline"
+CONF_STRAPPING = "strapping"
```

Comment:
I wonder if we should go as far a to making it like this?
```suggestion
CONF_IGNORE_STRAPPING_WARNING = "ignore_strapping_warning"
```